src/dialects/mssql/data-types.js

```diff
@@ -142,6 +142,20 @@ module.exports = BaseTypes => {
     }
   }
 
+  class TIME extends BaseTypes.TIME {
+    constructor() {
+      super();
+
+      if (this._zoned) {
+        warn('MSSQL does not support TIME WITH TIMEZONE. Plain `TIME` will be used instead.');
+        this.options.zoned = undefined;
```

Comment:
The line is not a problem? Previously I remove this.options in constructor. I think it can cause an error

Reply:
this specific line can be removed (same for the other dialects)

Reply:
Okay